### DISCUSSION TITLE
Delete reactions on others messages

### DISCUSSION THREAD

**Discussion by Perflyst** [victornpb/undiscord#343] [3. Ideas]
**Is your feature request related to a problem? Please describe.**
If someone wants to remove / clean his discord history and account he probably also wants to get rid of his reactions

**Describe the solution you'd like**
The script should also remove reactions on other peoples messages

**Comment by victornpb** [comment]
Sorry for the late reply, it is something that more ppl probably be interested, but I have to investigate how to do it
